packages/core/src/agent/index.ts

```diff
@@ -776,6 +778,14 @@ export class Agent<
     };
   }
 
+  async generate<Z extends ZodSchema | JSONSchema7 | undefined = undefined>(
+    messages: string | string[] | CoreMessage[],
+    args?: AgentGenerateOptions<Z> & { output?: 'text'; experimental_output?: undefined },
```

Comment:
This means you can't set output text + experimental_output

Reply:
shouldn't it be never?
```suggestion
    args?: AgentGenerateOptions<Z> & { output?: 'text'; experimental_output?: never},
```